frontend/appflowy_flutter/packages/flowy_infra/lib/colorscheme/default_colorscheme.dart

```diff
@@ -51,7 +50,8 @@ class DefaultColorScheme extends FlowyColorScheme {
           tint9: _lightTint9,
           main1: _lightMain1,
           main2: const Color(0xff00b7ea),
-          shadow: _black,
+          //15% black
+          shadow: const Color(0x26000000),
```

Comment:
Couldn't you use `_black.withOpacity(0.15)`? 
> https://api.flutter.dev/flutter/dart-ui/Color/withOpacity.html

Reply:
Maybe `Color.fromRGBO(0, 0, 0, 0.15)` is easilier readable than the hexadecimal value of `0.15`.

Reply:
Nice! I didn't know that and just changed it. Thanks @Xazin